[2024-03-27T04:03:55.334+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Process_W3_Data.create_directories manual__2024-03-27T04:03:52.146465+00:00 [queued]>
[2024-03-27T04:03:55.346+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Process_W3_Data.create_directories manual__2024-03-27T04:03:52.146465+00:00 [queued]>
[2024-03-27T04:03:55.347+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-27T04:03:55.375+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): create_directories> on 2024-03-27 04:03:52.146465+00:00
[2024-03-27T04:03:55.386+0000] {standard_task_runner.py:60} INFO - Started process 709 to run task
[2024-03-27T04:03:55.391+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Process_W3_Data', 'create_directories', 'manual__2024-03-27T04:03:52.146465+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/w3c.py', '--cfg-path', '/tmp/tmphuxlmp5h']
[2024-03-27T04:03:55.397+0000] {standard_task_runner.py:88} INFO - Job 13: Subtask create_directories
[2024-03-27T04:03:55.496+0000] {task_command.py:423} INFO - Running <TaskInstance: Process_W3_Data.create_directories manual__2024-03-27T04:03:52.146465+00:00 [running]> on host 94418fe46b42
[2024-03-27T04:03:55.678+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Process_W3_Data' AIRFLOW_CTX_TASK_ID='create_directories' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T04:03:52.146465+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-27T04:03:52.146465+00:00'
[2024-03-27T04:03:55.686+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-27T04:03:55.705+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Process_W3_Data, task_id=create_directories, execution_date=20240327T040352, start_date=20240327T040355, end_date=20240327T040355
[2024-03-27T04:03:55.767+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-27T04:03:55.797+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
